routes/weather.ts

```diff
@@ -260,10 +254,13 @@ export const getWateringData = async function( req: express.Request, res: expres
 	// Continue with the weather request
 	let timeData: TimeData = getTimeData( coordinates );
 	let wateringData: WateringData;
-	if ( local.useLocalWeather() ) {
-		wateringData = await getLocalWateringData( coordinates );
-	} else if ( adjustmentMethod !== 0 ) {
-		wateringData = await weatherProvider.getWateringData(coordinates);
+	if ( adjustmentMethod !== ADJUSTMENT_METHOD.MANUAL ) {
+		if ( !weatherProvider.getWateringData ) {
+			res.send( "Error: selected WeatherProvider does not support getWateringData" );
+			return;
+		}
+
+		wateringData = await weatherProvider.getWateringData( coordinates );
```

Comment:
I've just checked App and it is allowable to have `ADJUSTMENT_METHOD.MANUAL` as well as `checkCaliforniaRestrictions` in force at the same time. So I think we might need to be calling `getWateringData` even when we have MANUAL mode. This would explain why we had the confusion a few days ago.

@rayshobby , can you confirm whether the UI is correct and we do need to check California Restrictions even when in Manual mode or do we need to change App?

Reply:
@PeteBa That's exactly right and I completely missed that during the discussions with Ray. Weather restrictions do not depend on a weather adjustment method being set.

Reply:
This is fixed in #49